docs/source/streamflow-public-testnet.md

```diff
@@ -211,8 +211,116 @@ The following [script](https://github.com/livepeer/pm-params-calculator) can use
 - The value received per hour (in terms of ticket expected value)
 - The frequency of winning tickets (in terms of hours)
 
+Find below an example of using the script to project the value received per hour and frequency of winning tickets when transcoding 10 streams into 240p, 360p and 720p renditions:
+
+```
+➜  pm-params-calculator git:(master) python3 calc.py -f
+DEFAULTS
+---------
+Ticket redemption gas cost: 100000
+Transaction cost overhead: 0.01
+
+
+Enter the desired ticket expected value (gwei): 1000
+
+
+Enter the gas price (gwei) to use for ticket redemption transactions: 5
+Ticket redemption gas price: 5.0 gwei
+
+
+Transaction cost to redeem a ticket: 0.0005000000 ether ($0.1000000000)
+Ticket face value: 0.0500000000 ether ($10.0000000000)
+Ticket winning probability: 0.000020000000000
+1 out of 49999 tickets will win
+
+
+Enter the price per pixel (wei) to charge: 1000
+Price per pixel: 1000 wei
+
+
+Would you like to add a rendition to be encoded? (y/n): y
+Enter the output width: 426
+Enter the output height: 240
+Enter the output FPS (frames per second): 30
+Ether the number of streams of this renditions: 10
```

Comment:
There seems to be a typo here in the statement from the script itself `Ether -> Enter`

Reply:
Fixed in 5eff245